client/app/scripts/superdesk/itemList/itemList.js

```diff
@@ -0,0 +1,282 @@
+(function() {
+
+'use strict';
+
+var DEFAULT_OPTIONS = {
+    endpoint: 'search',
+    pageSize: 25,
+    page: 1,
+    sort: [{versioncreated: 'desc'}]
+};
+
+angular.module('superdesk.itemList', [])
+.service('itemListService', ['api', function(api) {
+    var getQuery = function(options) {
```

Comment:
it would be better like `function getQuery()`, seems like there is no need to assign it